lib/services/groups/groupService.js

```diff
@@ -27,6 +27,7 @@ var async = require('async'),
     iotManagerService = require('./../common/iotManagerService'),
```

Comment:
An entry in the CHANGES_NEXT_RELEASE regarding the changes introduced by this PR should be added (I guess that mentioning issue #596)